hyperdome_client/hyperdome_client.py

```diff
@@ -204,9 +214,9 @@ def send_message(self):
                 self.chat_history.append("You: " + message)
                 self.on_history_added()
                 if self.server.is_therapist: # needs auth
-                    session.post(f"{self.server.url}/message_from_therapist",headers={"username":self.server.username, "password":self.server.password,"message":message})
+                    self.session.post(f"{self.server.url}/message_from_therapist",headers={"username":self.server.username, "password":self.server.password,"message":message})
                 else: # normal user
-                    session.post(self.server.url + '/message_from_user', data = {'message':message, 'guest_id':self.uid} )
+                    self.session.post(self.server.url + '/message_from_user', data = {'message':message, 'guest_id':self.uid} )
```

Comment:
```suggestion
                    self.session.post(f'{self.server.url}/message_from_user', data={'message':message, 'guest_id':self.uid} )
```
fstrings are generally faster and easier to read than string concatenation